inst/pages/11_taxonomic_information.qmd

```diff
@@ -277,133 +277,40 @@ tse_merged
-and used to improve compatibility with assumptions related to specific
-statistical methods, mitigate biases, enhance the comparability of
-samples or features, or to obtain more interpretable values.
-
-Examples include the logarithmic transformation, calculation of
-relative abundances (percentages), and compositionality-aware
-transformations such as the centered log-ratio transformation (clr).
-
-Let us summarize some commonly used transformations in microbiome data
-science; further details and benchmarkings available in the
-references.
-
- * 'relabundance' relative transformation; also known as total sum
-   scaling (TSS) and compositional transformation. This converts
-   counts into percentages (at the scale [0, 1]) that sum up to
-   1. Much of the currently available taxonomic abundance data from
-   high-throughput assays (16S, metagenomic sequencing) is
-   compositional by nature, even if the data is provided as counts
-   [@Gloor2017].
- 
- * 'clr' Centered log ratio transformation [@Aitchison1986] is used to
-   reduce data skewness and compositionality bias in relative
-   abundances, while bringing the data to the logarithmic scale. This
-   transformation is frequently applied in microbial ecology
-   [@Gloor2017]. However, this transformation only applies to positive
-   values. Usual solution is to add pseudocount, which adds another
-   type of bias in the data. The robust clr transformation ('rclr')
-   aims to circumvent the need to add a pseudocount. While the
-   resulting values from these transformations are difficult interpret
-   directly, this transformation may enhance comparability of relative
-   differences between samples. It is part of a broader Aitchison
-   family of transformations; the additive log ratio transformation
-   (`alr') is also available. The robust clr ("rclr") is similar to
-   regular clr (see above) but allows data with zeroes and avoids the
-   need to add pseudocount [@Keshavan2010, @Martino2019].
-
-  * 'pa' presence/absence transformation ignores abundances and only
-   indicates whether the given feature is detected above the given
-   threshold (default: 0). This simple transformation is relatively
-   widely used in ecological research. It has shown good performance
-   in microbiome-based classification performance [@Giliberti2022,
-   Karwowska2024].
-
- * 'z' Z transformation scales data to zero mean and unit variance;
-   this us used to bring features (or samples) to more comparable
-   levels in terms of mean and scale of the values. This can enhance
-   visualization and interpretation of the data
-
-* 'log', 'log2', 'log10' Logarithmic transformations; used e.g. to
-   reduce data skewness; with compositional data the `clr` (or `rclr`)
-   transformation is often preferred.
-   
- * 'hellinger' Hellinger transformation equals to the square root of
-   relative abundances. This ecological transformation can be useful
```

Comment:
Why there is so many lines deleted?

Reply:
No, they are not. I only focused on the `ggsignif` library.

Reply:
These seem to be deleted in the PR as they are red?

These lines I just added a few days ago, they should remain in the book. If they were also duplicated then the duplicates can be removed of course.